### ISSUE TITLE
Driver crash can reset entire app

### ISSUE THREAD

**Issue by epsilon-a** [Maassoft/ColorControl#64]
I've had this happen a number of times, GPU driver crash from unstable OC or some other problem can cause the settings to corrupt, is there a way to keep them backed up?

**Comment by Maassoft** [comment]
Yes, I've had this problem as well a couple of times, especially when the system BSODs just after the config files have been written. Somehow they were zeroed out and unsalvageable.
The app should make a backup of most files now though, after successfully loading a .json file. Maybe there are still .backup files in this folder: C:\Users\username\AppData\Roaming\Maassoft\ColorControl
